Task_HelloWorld/index.js

```diff
@@ -1,26 +1,52 @@
-// Print hello World
-// Write your code here
-
-// Write your code above
-
-//Declare a variable i and set its value to 5
-// Write your code here
-
-// Write your code above
```

Comment:
The condition of the for loop is incorrect. It should be 'i < 5' instead of 'i > 5'